_sass/elements/_links.scss

```diff
@@ -28,20 +33,40 @@ a,
   &.button-secondary {
     border: none;
   }
-  img {
+  img, i, span {
     border: none;
   }
 
 }
 .link-beta {
-  @extend a;
+  @extend %link;
   color: $white;
   &:active,
   &:hover {
     text-decoration: none;
-    border-bottom: 1px dotted $white;
-    img{
+    border-bottom: $thin-border-size dotted $white;
+    img, i, span{
       border: none;
     }
   }
 }
+
+.link-active,
+.link-active-alpha {
+  @extend .link-alpha;
+  border-bottom: $thin-border-size dotted $base-link-color;
+}
+
+.link-active-beta {
+  @extend .link-beta;
+  border-bottom: $thin-border-size dotted $white;
+}
+
+a.link-no_under {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `link-no_under` should be written in lowercase with hyphens